meson.build

```diff
@@ -1823,8 +1855,1355 @@ executable('networkctl',
                         libshared],
            install : true)
 
-executable('test-sizeof', 'src/test/test-sizeof.c',
-           include_directories : includes)
+############################################################
+
+test_device_nodes = executable(
+    'test-device-nodes',
+    'src/test/test-device-nodes.c',
```

Comment:
Wouldn't it be nicer to move the tests into a separate src/test/meson.build file resp. into the meson.build of the corresponding directory.

One thing I didn't particularly like about the non-recursive make was that we had a single, huge Makefile.am